### PR TITLE
add Checks if `value` is classified as a `BigInt` primitive.

### PR DESCRIPTION
**Pull request by xgqfrms**
add `isBigInt`

### CODE REVIEW

File: isBigInt.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * isBigInt(BigInt('1'))
+ * // => true
+ *
+ * isBigInt(123)
+ * // => false
+ */
+function isBigInt(value) {
+  const type = typeof value
+  return type === 'bigint'
```
Comment (aashutoshrathi) [inline isBigInt.js:21]: Should we use our inbuilt files like getTag.
And use type using it, like this:
```js
return getTag(value) === '[object BigInt]'
```

Comment (xgqfrms) [reply isBigInt.js:21]: 👍 OK, I will modify it.

File: isBigInt.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * isBigInt(BigInt('1'))
+ * // => true
+ *
+ * isBigInt(123)
+ * // => false
+ */
+
+function isBigInt(value) {
+  const tag = getTag(value)
```
Comment (aashutoshrathi) [inline isBigInt.js:23]: Not sure if this variable is needed.
Rest LGTM ✅

Comment (xgqfrms) [reply isBigInt.js:23]: Maybe, you are right!
I just want to reuse the `tag` for the future extension.

Comment (xgqfrms) [reply isBigInt.js]: Thanks for your code review.

**Comment by falsyvalues** [comment]
Hi @xgqfrms!

Please clean up commits.

**Comment by xgqfrms** [comment]
@falsyvalues 

Sorry, I'm not sure what should be clean.

It's that mean to my previous commits history？

**Comment by aashutoshrathi** [comment]
I think you need to squash up your commits, specifically the ones concerning `label.yml` and `README.md` as there is no change in them in the end.

**Pull request closed without merging**